include/common/common/utility.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <spdlog/fmt/bundled/core.h>
+
+#include "range/v3/view/transform.hpp"
+#include <range/v3/algorithm/copy.hpp>
+#include <range/v3/range.hpp>
+
+#include <utility>
+
+namespace utility
+{
+
+// Replacement for yet unimplemented C++23 to_underlying
+constexpr auto
+toUnderlying( auto val ) -> std::underlying_type_t<decltype( val )>
+{
+    return static_cast<std::underlying_type_t<decltype( val )>>( val );
+}
+
+// clang-format off
+template <typename T>
+concept convertibleToCStrRange = requires ()
+{
+  requires ranges::range<T>;
+  requires requires (ranges::range_value_t<T> a) {{ a.c_str() } -> std::convertible_to<const char*>;}; 
+};
+// clang-format on
+
+template <typename T>
+constexpr auto
+convertToCStrVector( T&& range )
+    requires ranges::contiguous_range<T> && std::convertible_to<ranges::range_value_t<T>, const char*>
+{
+    return range | ranges::views::transform( []( auto&& a ) { return static_cast<const char*>( a ); } ) |
+        ranges::to_vector;
+}
+
```

Comment:
Leave comment according to our code style.

Reply:
You mean comment at the end of function that duplicate the name of function? If so, it should be used for every function, even for so short one?

Reply:
I mean brief description of function.
And yep, for every function we should leave comment with its name.